docs/static/ds6/docs.js

```diff
@@ -1,22 +1,22 @@
-$_mod.installed("@ebay/skin$6.0.4","makeup-expander","0.4.0");
-$_mod.main("/makeup-expander$0.4.0","");
-$_mod.installed("makeup-expander$0.4.0","custom-event-polyfill","0.3.0");
+$_mod.installed("@ebay/skin$6.0.4","makeup-expander","0.5.0");
```

Comment:
Let's remove this generated docs.js file from the PR, as master branch is still on version 0.4.0 of expander (I only updated it in 6.1.0 branch).